app/assets/stylesheets/admin/loan_questions.scss

```diff
@@ -0,0 +1,45 @@
+.loan-questions {
```

Comment:
Can we call all your stuff 'manage-loan-questions'. I worry about people getting confused between this and mine which is called 'questionnaire(s)'.